### PR TITLE
Clarify some Plague things

### LINKED ISSUE #606: Some plague additions
> plague disappears even if turned plague members remain (inconsistent)

> it isnt specified that PB cant be contaminated

@McTsts

### PR DESCRIPTION
**Pull request by ThomasRubycrust**
Fixes #606

### CODE REVIEW

File: other/plague/plague bearer [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 Contaminating a player is an immediate ability. Contaminated players are contagious, and remain so till their death.
-Any player who visits or is visited by a contagious player is also immediately contaminated. Any type of action (target, attack, investigation, etc) is counted as a visit.
+Any player who visits or is visited by a contagious player is immediately contaminated. Any type of action (target, attack, investigation, etc) is counted as a visit.
 At the start of the second night after being contaminated the player is informed that they are contaminated.
-At the end of the second night after being contaminated the player will stop being contagious and die (not an attack).
-If no members of the plague team remain all players are no longer contaminated, however they are not informed about this.
-If the Plague Bearer attempts to use a rodent on the Plague Doctor, the attempt fails, the Plague Doctor immediately dies (not an attack) and the rodent is *not* used up.
+At the end of the second night after being contaminated the player will cease being contagious and die. This is not an attack.
```
Comment (venomousbirds) [inline other/plague/plague bearer:12]: don't think 'cease' sounds good here

Comment (McTsts) [reply other/plague/plague bearer:12]: I agree

Comment (venomousbirds) [reply other/plague/plague bearer]: changed to stop, sounds much better

Comment (McTsts) [reply other/plague/plague bearer]: That was the original wording anyway, not sure why @cryptidjayden changed it.

**Review by McTsts — APPROVED** [review]

**Pull request merged**